src/current/_includes/cockroachcloud/quickstart/create-a-free-cluster.md

```diff
@@ -3,7 +3,11 @@
 1. If you haven't already, <a href="https://cockroachlabs.cloud/signup?referralId={{page.referral_id}}" rel="noopener" target="_blank">sign up for a CockroachDB {{ site.data.products.cloud }} account</a>.
 1. [Log in](https://cockroachlabs.cloud/) to your CockroachDB {{ site.data.products.cloud }} account.
 1. On the **Clusters** page, click **Create Cluster**.
-1. On the **Create your cluster** page, select **Serverless**.
-1. Click **Create cluster**.
+1. On the **Select a plan** page, select **Serverless**.
+1. On the **Cloud & Regions** page, select a cloud provider (GCP or AWS) in the **Cloud provider** section. 
+1. In the **Regions** section, select a region for the cluster. Refer to [CockroachDB {{ site.data.products.cloud }} Regions]({% link cockroachcloud/regions.md %}) for the regions where CockroachDB {{ site.data.products.serverless }} clusters can be deployed. To create a multi-region cluster, click **Add region** and select additional regions. A cluster can have at most six regions.
```

Comment:
Can you double check that the max number of regions for the Serverless free cluster is six? I'm seeing three supported regions so far.

Reply:
We only have three serverless regions enabled in staging 😄

Reply:
ahh okay, it's a staging thing.